umm-spec-lib/src/cmr/umm_spec/validation/umm_spec_collection_validation.clj

```diff
@@ -63,24 +51,12 @@
    :MetadataAssociations (vu/unique-by-name-validator metadata-association-name)
    :TilingIdentificationSystems tiling-identification-system-validations})
 
-(defn- temporal-end-date-in-past-validator
-  "Validate that the end date in TemporalExtents is in the past"
-  [field-path value]
-  (when (and value (not (date/is-in-past? value)))
-    {field-path [(str "Ending date should be in the past. Either set ending date to a date "
-                      "in the past or remove end date and set the ends at present flag to true.")]}))
-
-(def temporal-extent-warning-validation
-  {:RangeDateTimes (v/every {:BeginningDateTime vu/date-in-past-validator
-                             :EndingDateTime temporal-end-date-in-past-validator})
-   :SingleDateTimes (v/every vu/date-in-past-validator)})
-
 (def collection-validation-warnings
  "Defines validations for collections that we want to return as warnings and not
  as failures"
  {:RelatedUrls (v/every url/related-url-validations)
   :Projects project/projects-warning-validation
-  :TemporalExtents (v/every temporal-extent-warning-validation)
+  :TemporalExtents (v/every temporal-extent/temporal-extent-warning-validation)
```

Comment:
If you pass in the strict validation header do we return failures if there is any warning? I don't see a test for it, which we should probably add if that's how it works.

Reply:
Are you talking about the validate-umm-c header? If so, there is a test in collection-umm-spec-validation-test (cmr.system-int-test.ingest.collection-umm-spec-validation-test) that tests that the warnings are returned as errors when the header is true.

Reply:
I think that's the header - so this warning would be treated as an error in that case? I'm ok with it if there's existing tests, I just wasn't sure of the behavior.

Reply:
Yes, in that case the warning would be treated as an error. There is a test for the header, but not specific to every validation warning.